xknx/io/request_response/tunnelling.py

```diff
@@ -47,15 +49,24 @@ async def send_request(self) -> None:
         self.transport.send(self.create_knxipframe(), addr=self.data_endpoint_addr)
 
     def create_knxipframe(self) -> KNXIPFrame:
-        """Create KNX/IP Frame object to be sent to device."""
-        cemi = CEMIFrame.init_from_telegram(
-            telegram=self.telegram,
-            code=CEMIMessageCode.L_DATA_REQ,
-            src_addr=self.src_address,
-        )
+        """Create KNX/IP Frame object to be sent on bus."""
+        if self.telegram.tpdu_type == TPDUType.T_DATA:
+            pdu: CEMIFrame | TPDU = CEMIFrame.init_from_telegram(
+                telegram=self.telegram,
+                code=CEMIMessageCode.L_DATA_REQ,
+                src_addr=self.src_address,
+            )
+        else:
+            # all other tpdu_types are non CEMI types
+            pdu = TPDU.init_from_telegram(
+                None,  # TODO: Remove this :)
```

Comment:
We need to remove this from the TPDU class completely. xknx is not needed there anymore.